calculate_radius_max_wind

```diff
@@ -0,0 +1,60 @@
+def calculate_radius_max_wind(rho_air, latitude, max_wind_speed, central_pressure,
+                              ambient_pressure, radius_30kts, radius_50kts):
+    """
+    INPUT:
+    - *rho_air* (float) Air density.
+    - *ambient_pressure* (float) Ambient pressure.
+    - *latitude* (flloat) Latitude of center of storm.
+    - *max_wind_speed* (float) Speed of maximum wind.
+    - *central_pressure* (float) Central pressure of storm.
+    - *radius_30kts* (float) Maximum radius of 30 kts wind.
+    - *radius_50kts* (float) Maximum radius of 50 kts wind.
+
+
+    OUTPUT:
+    - *radius* (float) Radius of maximum wind.
```

Comment:
So this only computes it for one value?  Maybe we should have the ability to compute for every time point in a `storm` object?